emark/backends.py

```diff
@@ -1,60 +1,78 @@
-import copy
-import smtplib
 import uuid
 
-from django.conf import settings
+from django.core.mail import EmailMessage
 from django.core.mail.backends.console import EmailBackend as _EmailBackend
 from django.core.mail.backends.smtp import EmailBackend as _SMTPEmailBackend
-from django.core.mail.message import sanitize_address
 
 from emark import models
 from emark.message import MarkdownEmail
 
 __all__ = [
     "ConsoleEmailBackend",
+    "SMTPEmailBackend",
     "TrackingConsoleEmailBackend",
     "TrackingSMTPEmailBackend",
 ]
 
 
+class RenderEmailBackendMixin:
+    def send_messages(self, email_messages):
+        self._messages_sent = []
+        for message in email_messages:
+            if isinstance(message, MarkdownEmail):
+                message.render()
+        try:
+            return super().send_messages(email_messages)
+        finally:
+            models.Send.objects.bulk_create(self._messages_sent)
```

Comment:
We don't want to track sends outside of track backend, right? So I think we should remove these lines from here.
```suggestion
```